jenkins/run_aggregation_broker.groovy

```diff
@@ -25,7 +25,7 @@ pipeline{
                         args += "-p ${params.db_password} "
                     }
 
-                    if(!utils.runScript("/logan/bin/aggregation_broker ${args}") == 0){
+                    if(!utils.runScript("docker run --name agg_broker -d --network=host logan logan/bin/aggregation_broker ${args}") == 0){
```

Comment:
Change to run commands inside a docker container, instead of actually interacting with docker run args